tools/cli/commands/changelog.js

```diff
@@ -390,16 +412,59 @@ async function changelogArgs( argv ) {
 				argv.args.push( argv.ver );
 			}
 			break;
+		case 'squash':
+			if ( argv.r && argv.readme ) {
+				readme = true;
+			} else {
+				readme = await promptReadme( argv );
+			}
+			argv.args = [ 'squash' ];
+			break;
 	}
 
-	// Remove project from arguments list we pass to the changelogger.
+	// Remove the project from the list of args we're passing to changelogger.
+	argv = await removeArgs( argv, removeArg );
+
+	// Run the changelogger command.
+	await changeloggerCli( argv );
+
+	// Squash just the readme if necessary.
+	if ( readme ) {
+		readmeSquash( argv );
+	}
+
+	// Add any newly added changelog files.
+	await gitAdd( argv );
+
+	console.log( chalkJetpackGreen( argv.success ) );
+}
+
+/**
+ * Handles squashing just the readme file.
+ *
+ * @param {object} argv - arguments passed as cli.
+ */
+async function readmeSquash( argv ) {
+	console.log( 'Updating readme...' );
+	await runCommand( 'tools/plugin-changelog-to-readme.sh', `${ argv.project }` );
```

Comment:
runCommand takes an array of args.
```suggestion
	await runCommand( 'tools/plugin-changelog-to-readme.sh', [ `${ argv.project }` ] );
```